### PR TITLE
SSC docs titles and content

### PR DESCRIPTION
**Pull request by s-santillan**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs

All content reviewed in internal drafts for SSC docs and Pricing and Billing.

### CODE REVIEW

File: docs/semgrep-sc/sc-getting-started.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+<ul id="tag__badge-list">
+{
+Object.entries(frontMatter).filter(
+    frontmatter => frontmatter[0] === 'tags')[0].pop().map(
+    (value) => <li class='tag__badge-item'>{value}</li> )
+}
+</ul>
+
+# Scanning open-source dependencies
```
Comment (pabloest) [inline docs/semgrep-sc/sc-getting-started.md:24]:
```suggestion
# Scanning open source dependencies
```

File: docs/semgrep-sc/sc-getting-started.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
+---
+slug: scanning-open-source-dependencies
+append_help_link: true
+description: "Scan your codebase's open-source dependencies with Semgrep Supply Chain's high-signal rules that determine a vulnerability's reachability."
```
Comment (pabloest) [inline docs/semgrep-sc/sc-getting-started.md:4]:
```suggestion
description: "Scan your codebase's open source dependencies with Semgrep Supply Chain's high-signal rules that determine a vulnerability's reachability."
```

File: docs/semgrep-sc/sc-getting-started.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+   <td>Type of tool</td>
+   <td>Static application security testing (SAST)</td>
+   <td>Software composition analysis (SCA)</td>
+  </tr>
+  <tr>
+   <td>Scan target
+   </td>
+   <td>First-party code (your codebase or repository)
+   </td>
+   <td>Open-source dependencies
```
Comment (pabloest) [inline docs/semgrep-sc/sc-getting-started.md:70]:
```suggestion
   <td>Open source dependencies 
```

File: docs/semgrep-sc/sc-getting-started.md [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+</table>
+
+## Language support
+
+Refer to [Supported languages](/docs/semgrep-sc/supply-chain-supported-languages) to see all languages supported by Semgrep Supply Chain.
+
+## Limitations of Semgrep Supply Chain scans
+
+* Semgrep Supply Chain supports scanning for transitive or indirect dependencies for all of its supported languages **except Java**. Findings are collected and displayed in **Semgrep App > Supply Chain**.
+* Semgrep Supply Chain detects transitivity only for **Python and Javascript **languages.
```
Comment (pabloest) [inline docs/semgrep-sc/sc-getting-started.md:117]:
```suggestion
* Semgrep Supply Chain detects transitivity only for **Python and JavaScript **languages.
```

File: docs/semgrep-sc/sc-getting-started.md [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+See [Ignoring dependency findings](/docs/semgrep-sc/ignoring-lockfiles-dependencies).
+
+## Triaging and remediating dependency findings
+
+Semgrep Supply Chain enables developers to perform triage and remediation through the **Dependencies** page. On this page you can perform the following:
+
+* View specific **reachable** vulnerable lines of code in your codebase. This helps to evaluate the threat.
+* View specific lines of code where your dependency is being declared.
+* Triage a dependency finding.
+* Attach a PR or Jira ticket to the finding.
```
Comment (pabloest) [inline docs/semgrep-sc/sc-getting-started.md:201]:
```suggestion
* Attach a PR or MR, or Jira ticket to the finding.
```

ok to add MR?

File: docs/semgrep-sc/sc-glossary.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+Semgrep Supply Chain uses manifest files to resolve transitive dependencies for some languages. For more information, see [Supported languages](/docs/semgrep-sc/supply-chain-supported-languages).
+
+## Package manager
+
+A software tool that interacts with a package registry to download, upload, or search for dependencies. Package managers typically generate lockfiles by analyzing manifest files.
+
+## Package registry
+
+A package registry stores dependencies and provides a means to upload or download dependencies. Each programming language has its own separate registry such as npm for Javascript and PyPi for Python.
```
Comment (pabloest) [inline docs/semgrep-sc/sc-glossary.md:44]:
```suggestion
A package registry stores dependencies and provides a means to upload or download dependencies. Each programming language has its own separate registry such as npm for JavaScript and PyPI for Python.
```

File: docs/semgrep-sc/sc-supported-langs.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+  <tr>
+   <td>r2c rule-writing support
+   </td>
+   <td>Quickly release new rules for all critical and high vulnerabilities based on the latest <a href="https://nvd.nist.gov/vuln">security advisories</a>.
+   </td>
+   <td>No commitment for new rules based on the latest security advisories.
+   </td>
+  </tr>
+  <tr>
+   <td>Semgrep SAST language support
```
Comment (pabloest) [inline docs/semgrep-sc/sc-supported-langs.md:135]: Should we refer to this as “Semgrep engine support?”

File: docs/semgrep-sc/sc-triage-remediation.md [hunk 8]
```diff
@@ -0,0 +1,220 @@
...
+  <tbody><tr>
+   <td>Exposure
+   </td>
+   <td>Filters are based on the <a href="/docs/semgrep-sc/sc-glossary">reachability</a> of a vulnerability. <strong>Reachable findings</strong> are displayed by default.
+   </td>
+  </tr>
+  <tr>
+   <td>Severity
+   </td>
+   <td>Filters are based on the severity of a vulnerability. Semgrep Supply Chain rules use severity values set by GitHub Advisory Database.
```
Comment (pabloest) [inline docs/semgrep-sc/sc-triage-remediation.md:121]:
```suggestion
   <td>Filters are based on the severity of a vulnerability. Semgrep Supply Chain rules use severity values set by the GitHub Advisory Database.
```

**Review by pabloest — APPROVED** [review]
I couldn’t leave this as a comment so I’m writing it here.

Is it possible to make `static/img/sc-reachability-analysis.jpg` as a PNG instead of a JPG? It would probably be smaller in file size and still of high quality.

File: docs/semgrep-sc/sc-glossary.md [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+A reachable finding means that you are using both a vulnerable piece of code (the **usage**) and the vulnerable version of a dependency. Within Semgrep Supply Chain, specific findings (usages) are grouped together by their vulnerability.
+
+Continuous integration scans with Semgrep Supply Chain rules block any merge or pull requests upon detecting any reachable findings.
+
+See also [Reachability](#reachability).
+
+
+## Reachability
+
+Reachability refers to whether or not a vulnerable piece of code from a dependency is used in the codebase itself. In Semgrep Supply Chain, both a dependency's vulnerable version and code pattern must match before a vulnerability is considered reachable.
```
[the remark this answers is not in the document]
Comment (s-santillan) [reply docs/semgrep-sc/sc-glossary.md:57]: I'm going with, "... vulnerable version and code pattern must match for a vulnerability to be considered reachable."

File: docs/semgrep-sc/sc-supported-langs.md (hunk 7 above)
Comment (s-santillan) [reply docs/semgrep-sc/sc-supported-langs.md:135]: Sure. And this bit is going into my bucket of thoughts on "how do we differentiate sast/engine/ssc".

**Comment by s-santillan** [comment]
> I couldn’t leave this as a comment so I’m writing it here.
> 
> Is it possible to make `static/img/sc-reachability-analysis.jpg` as a PNG instead of a JPG? It would probably be smaller in file size and still of high quality.

Fixed in latest. Merging.

**Pull request merged**